main.nf

```diff
@@ -29,7 +29,13 @@ params.center = "ALL"
 params.create_new_maf_db = false
 // release name (pass in TEST.public to test the public release scripts)
 params.release = "TEST.consortium"
-
+// List all centers to be processed for maf_process
+params.maf_centers = "ALL"
+maf_centers = params.maf_centers?.split(",").toList()
```

Comment:
It's a bit confusing to have `ch_maf_centers` (later) and then `maf_centers` and then `maf_center_list` here and later have all three in the function `process_maf_helper` , could you distinguish the three variables more clearly? 

Also it feels like `maf_center_list` should be an immutable global static list, perhaps we shouldn't be modifying it later like `maf_center_list = maf_centers`

Maybe `maf_centers` could be `input_maf_centers`

Reply:
The `maf_centers` is a list converted from user input so the user can feed `params.maf_centers` as a comma-separated list (a,b,c) instead of space-separated list (a b c). Otherwise, `params.maf_centers a,b,c ` will be treated as one center named `a,b,c` instead of 3 centers. 
The `maf_center_list` is the default list for all centers which will be updated whenever we are not using `ALL` as the value of `params.maf_centers`.  Basically, user can feed `params.maf_centers` as `ALL` or a center list. I just pushed a change and let me know if it's more clear to you.

Reply:
Yup, just wanted that clarity in the code say someone needs to modify / look at it in the future months from now given the very similar naming. Thanks!